client/src/containers/ResetPassword/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { SyntheticEvent, useState } from 'react';
+import * as actions from '../LoginPage/logic/actions';
+import { useDispatch } from 'react-redux';
+import { Button, Divider, Form, Grid, Header, List, Popup, Segment } from 'semantic-ui-react';
+import { useTranslation } from 'react-i18next';
+import styles from '../ForgotPassword/styles.module.scss';
+import { Link } from 'react-router-dom';
+import PasswordCheck from '../../components/PasswordCheck';
+import PasswordInput from '../../components/common/PasswordInput';
+
+interface Props {
+	match: {
```

Comment:
Use RouteComponentProps from react-router.
https://stackoverflow.com/questions/48138111/what-typescript-type-should-i-use-to-reference-the-match-object-in-my-props